docs-src/python/how-to-send-a-signal-from-a-client-in-python.md

```diff
@@ -0,0 +1,16 @@
+---
+id: how-to-send-a-signal-from-a-client-in-python
+title: How to send a Signal from a Client in Python
+sidebar_label: Send a Signal from a Client
+description: Send a Signal from a Client
+tags:
+  - developer-guide
+  - sdk
+  - python
+---
+
+To send a Signal to a Workflow Execution from Client code, use the [`signal()`](https://python.temporal.io/temporalio.workflow.html#signal) API in the Workflow interface.
```

Comment:
```suggestion
To send a Signal to a Workflow Execution from Client code, use the [`signal()`](https://python.temporal.io/temporalio.workflow.html#signal) method on the Workflow handle.
```

Also, this assumes that the user at this point knows what a "handle" is. If not, may need to clarify.